### PR TITLE
Update the user manual with the command publish

### PR DESCRIPTION
**Pull request by Isotech42**
I added a few lines about how to use the `publish` command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,12 @@ La commande `statique` permet de générer un site web statique à partir d'un f
     `init <PATH>` : Initialise le projet du web statique.\
     `build <PATH>` : Construit le site web statique à partir du fichier Markdown.\
     `serve <PATH> <PORT>` : Lance le serveur web statique construit.\
-    `clean <PATH>` : Supprime les fichiers générés.
+    `clean <PATH>` : Supprime les fichiers générés.\
+    `publish <PATH> <remote_path> <ssh_url>` : Publie le site web statique sur un dépôt GIT en SSH.
```
Comment (clemdiep-heig) [inline README.md:48]: It's not a git repo but it only copy the file to the remote path

File: README.md [hunk 2]
```diff
@@ -44,7 +44,12 @@ La commande `statique` permet de générer un site web statique à partir d'un f
...
+
+Précisions pour la commande `publish`:\
+Le paramètre `<ssh_url>` s'écrit de la même manière qu'une URL SSH classique, ce qui permet d'écrire simplement `url` ou `user@url`.\
+Par défaut, l'application récupère les clés SSH dans `~/.ssh/id_rsa`, cependant, il est possible d'ajouté l'option `-p` ou `--password` afin d'utiliser un mot de passe (celui-ci étant demandé **après** avoir lancé la commande).
```
Comment (clemdiep-heig) [inline README.md:52]:
```suggestion
Par défaut, l'application récupère les clés SSH dans `~/.ssh/id_rsa`, cependant, il est possible d'ajouter l'option `-p` ou `--password` afin d'utiliser un mot de passe (celui-ci étant demandé **après** avoir lancé la commande).
```

**Review by clemdiep-heig — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Isotech42) [reply README.md]: I update the sentence, let me know if it's better.

**Review by clemdiep-heig — APPROVED** [review]
Perfect 🍺

**Review by MordoJ — APPROVED** [review]
Perfect 👌

**Pull request merged**